joplin/base/models.py

```diff
@@ -77,6 +78,13 @@ class ServicePage(Page):
     subpage_types = []
     base_form_class = custom_forms.ServicePageForm
 
+    search_fields = Page.search_fields + [
```

Comment:
This doesn't actually work -- despite this being the implementation suggested by the docs. I might need to reindex or something. I left it in the code as a starting point.